webapp/security/views.py

```diff
@@ -435,13 +423,29 @@ def cve_index():
         "pending": {"name": "Vulnerable", "icon": "warning"},
         "ignored": {"name": "Ignored", "icon": "error-grey"},
         "released": {"name": "Fixed", "icon": "success"},
+        "vulnerable": {"name": "Vulnerable", "icon": "warning"},
     }
 
+    for cve in high_priority_cves:
+        cve["summarized_status"] = {}
+        get_summarized_status(
```

Comment:
I see that `get_summarized_status()` sets the `cve["summarized_status"]` within the function. Can we modify it so that it returns the value of the summarized_status and we set it where the function is being called? Since it is a getter function I think it should be more explicit that the function returns the value of the status rather than setting it implicitly.

Something like this:

```suggestion
    for cve in high_priority_cves:
        cve["summarized_status"] = get_summarized_status(...)
```

Reply:
Nice, this is better indeed!